dist/PathTools/META.json

```diff
@@ -9,7 +9,7 @@
       "perl_5"
    ],
    "meta-spec" : {
-      "url" : "http://search.cpan.org/perldoc?CPAN::Meta::Spec",
+      "url" : "https://metacpan.org/pod/CPAN::Meta::Spec",
```

Comment:
These META.json files are generated by EUMM so it's pointless to change them.

Reply:
But in this case (and the one below), *we* control the code (because "blead-upstream").  So how should these changes be made?

Reply:
It's not a matter of control but that this will have no effect - these files are regenerated when making a release. This has to be fixed in EUMM or CPAN::Meta.

Reply:
> It's not a matter of control but that this will have no effect - these files are regenerated when making a release. This has to be fixed in EUMM or CPAN::Meta.

When you say *"making a release"*, do you mean making a release of Perl 5 or a release of one of those distros separately on CPAN?

Reply:
I mean releases of these distros on CPAN. The META.json files here have no bearing on Perl itself.

Reply:
Suppose I were to proceed as if I were the person designated to do a CPAN release of PathTools.  I would create a separate git repository for it and copy relevant files from blead (with the exception of files such as `.gitignore`, `MANIFEST` and `MANIFEST.SKIP` (if any) that might need special handling).

Though I haven't been assigned to do a CPAN release of PathTools, I've done all of that locally (including updating `meta_spec:url` in `META.json` as proposed above) and pushed my work to github.  See:  https://github.com/jkeenan/PathTools/.

Now, I'll grant you that, in *my* repository for PathTools, when I say `perl Makefile.PL`, what is written to `Makefile` and to `MYMETA.json` is the search.cpan.org URL.
```
$ bleadperl Makefile.PL
Checking if your kit is complete...
Looks good
Generating a Unix-style Makefile
Writing Makefile for Cwd
Writing MYMETA.yml and MYMETA.json
[PathTools] 608 $ ack 'search\.cpan\.org' .
Makefile
635:	$(NOECHO) $(ECHO) '      "url" : "http://search.cpan.org/perldoc?CPAN::Meta::Spec",' >> META_new.json

MYMETA.json
12:      "url" : "http://search.cpan.org/perldoc?CPAN::Meta::Spec",
```
Furthermore, once I say `make dist` and create a CPAN-ready tarball and then examine that tarball, I see that `make dist` has changed the contents of `META.jsn`:
```
$ pwd
/tmp/PathTools-3.80
$ ack 'search\.cpan\.org' .
META.json
12:      "url" : "http://search.cpan.org/perldoc?CPAN::Meta::Spec",
```
These I would consider bugs (plural) in ExtUtils::MakeMaker.  (No evidence here as yet of bugs in CPAN::Meta::Spec.)

However, I think that's an argument for filing a bug report against ExtUtils::MakeMaker.  Has that been done?

I don't see that as a compelling argument for letting the old URL remain in `dist/PathTools/META.json` in core.

Thank you very much.
Jim Keenan

Reply:
Your assessment is correct. But there is no reason to edit a META.json by hand, at all, including this one.

Reply:
I agree it shouldn't be changed here. And as often in toolchain matters, there are some non-obvious difficulties in changing it (though I wouldn't say it's impossible).

Reply:
I will revert the changes to the two .json files, as it appears that we will have to wait for changes in toolchain.